services/gundeck/src/Gundeck/Env.hs

```diff
@@ -127,28 +129,9 @@ createRedisPool l endpoint identifier = do
     Log.msg (Log.val $ "starting connection to " <> identifier <> "...")
       . Log.field "connectionMode" (show $ endpoint ^. rConnectionMode)
       . Log.field "connInfo" (show redisConnInfo)
+  let connectWithRetry = Redis.connectRobust l (exponentialBackoff 50000) 
```

Comment:
To the reviewer: I wonder whether what currently is `exponentialBackoff 50000` should be more flexible and I can imagine two ways to add flexibility,
* `50000` is a magic number and should perhaps rather come from the environment variables, however, the choice of the constant might not be mattering much enough, since more importantly the back-off would always be exponential;
* `exponentialBackoff` is just one of many retry policies in the [retry](https://hackage.haskell.org/package/retry) package and perhaps retries should be limited in their number or by the time they require and/or even
  * combine this with environment configuration.